Adyen/Core/Core Protocols/Component.swift

```diff
@@ -14,10 +14,15 @@ extension Component {
 
     /// Finalizes the payment if there is any, after being processed by payment provider.
     /// - Parameter success: The status of the payment.
-    /// :nodoc:
-    public func finalizeIfNeeded(with success: Bool) {
-        (self as? FinalizableComponent)?.didFinalize(with: success)
+    /// - Parameter completion: The block to execute after the component finalize it's activity.
```

Comment:
- Parameter completion: The block to execute after the component finalize its activity.

Reply:
comment on the comment :P

Reply:
will fix in next PR :D